agenta-backend/agenta_backend/routers/variants_router.py

```diff
@@ -133,7 +133,7 @@ async def remove_variant(
 @router.put("/{variant_id}/parameters/")
 async def update_variant_parameters(
     variant_id: str,
-    payload: UpdateVariantParameterPayload,
+    payload: UpdateVariantParameterPayload = Body(...),
```

Comment:
Why do we have this?

Reply:
copied from another endpoint, can be removed actually